taskcluster/mozillavpn_taskgraph/transforms/beetmover_apt.py

```diff
@@ -43,6 +43,11 @@ def get_gcs_sources(dependent_task):
 
 @transforms.add
 def beetmover_apt(config, tasks):
+    is_production = (
+        config.params["level"] == "3" and config.params["tasks_for"] == "action"
+    )
+    bucket = "release" if is_production else "dep"
```

Comment:
Just to be 100% clear: the idea is that both the `promote` and `apt` tasks will push to the exact same debian repo (but with different package names)? If so, this block looks like it ought to be fine. (We'll be able to verify the `dep` path in the staging repo, but the `release` one obviously can only be fully tested in a real release.)

Reply:
So, my understanding is that `beetmover-promote` and `beetmover-ship` send packages to the production archive.mozilla.org site whenever they are triggered by shipit and we want to follow the same logic for the `beetmover-apt` task.

The promotion and shipping phases on archive.mozilla.org are distinguished by different directory paths (eg: promotion puts artifacts into the `candidates` directory). The objective of this PR is to distinguish promotion and shipping in the APT repository by changing the package name.

Reply:
Are we talking about uploading to archive.m.o, the debian repository, or both? (Both are handled by beetmover, but they are distinct things can be done or not done independently AFAIK.)